src/bridge/bridge.hpp

```diff
@@ -27,19 +27,79 @@
 #pragma once
 
 #define BRIDGE_USE_PIPES 1
+
 #include "ProtobufMessages.pb.h"
 #include <variant>
 #include <optional>
-#include "../VRDriver.hpp"
+#include <thread>
+#include <mutex>
+#include <queue>
 
 enum BridgeStatus {
     BRIDGE_DISCONNECTED = 0,
     BRIDGE_CONNECTED = 1,
     BRIDGE_ERROR = 2
 };
 
-BridgeStatus runBridgeFrame(SlimeVRDriver::VRDriver &driver);
+namespace SlimeVRDriver {
+    class VRDriver;
+
+    class Bridge {
```

Comment:
I'd *personally* prefer an abstract class with transport agnostic code, and derived classes that implement the transport specific bits. (This would allow for the possibility of multiple transports, fallbacks, etc. -- but it's kinda irrelevant right now unless we decide to put in effort to make unix sockets also work on windows... which we don't actually have a need for.)

this is really just a nitpick. don't block the PR on this.

Reply:
The code is over 6 months old there was no unix implementation :)